packages/ui/src/components/vuestic-components/va-input/VaInput.vue

```diff
@@ -188,14 +196,25 @@ export default {
       return { color: this.colorComputed }
     },
     containerStyles () {
-      return {
-        backgroundColor:
-          this.computedError ? (this.computeColor('danger') ? getHoverColor(this.computeColor('danger')) : '')
-            : this.c_success ? (this.computeColor('success') ? getHoverColor(this.computeColor('success')) : '') : '#f5f8f9',
-        borderColor:
-          this.computedError ? this.computeColor('danger')
-            : this.c_success ? this.computeColor('success')
-              : this.isFocused ? this.computeColor('dark') : this.computeColor('gray'),
+      if (!this.insidePagination) {
+        return {
+          minHeight: '2.375rem',
+          backgroundColor:
+            this.computedError ? (this.computeColor('danger') ? getHoverColor(this.computeColor('danger')) : '')
+              : this.c_success ? (this.computeColor('success') ? getHoverColor(this.computeColor('success')) : '') : '#f5f8f9',
+          borderColor:
+            this.computedError ? this.computeColor('danger')
+              : this.c_success ? this.computeColor('success')
+                : this.isFocused ? this.computeColor('dark') : this.computeColor('gray'),
+        }
+      } else {
+        const isPaginationFlat = this.$parent.$parent.flat // not really good :(
```

Comment:
If placed in app root - this will break horribly.

There should be ways to play around that.